lib/globals.ts

```
@@ -0,0 +1,18 @@
+      first: R,
+      other: P
+    ): R & P;
+  }
+}
+
+Object.merge = function (first, other) {
+  return merge(first, other);
+};
+
+// Ensure this file is treated as a module
+export {};
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)